.github/workflows/inspektor-gadget.yml

```diff
@@ -872,6 +876,19 @@ jobs:
           hash=$(sha256sum $i | cut -d' ' -f1)
           echo "${hash}  $(basename $i)" >> inspektor-gadget-${{ github.ref_name }}_checksums.txt
```

Comment:
I see this is from a prior commit (7080101) but this might get simplified as just:
```shell
sha256sum *-gadget-*-*-tar-gz/*-gadget-*-*.tar.gz inspektor-gadget-${{ github.ref_name }}.yaml > SUMS
```
(I'm struggling to find a link to the filename convention, but `SUMS` is seen regularly)